packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -52,7 +52,7 @@ export class OrgAuthInfo {
         }
       }
 
-      return JSON.stringify(defaultUsernameOrAlias).replace(/\"/g, '');
+      return defaultUsernameOrAlias;
```

Comment:
There are two copies of `getDefaultUsernameOrAlias()` (one in packages/salesforcedx-utils-vscode/src/config/configUtil.ts, and the other one here, in packages/salesforcedx-vscode-core/src/util/authInfo.ts)

Can they be consolidated?
Or if not consolidated, can they match each other?

Reply:
I feel like the two methods are different enough to remain separate since `OrgAuthInfo.getDefaultUsernameOrAlias()` is doing an extra check for the global default username and it's also displaying warning messages to the user. Would we want that to happen by default from `configUtil.ts`? If not, maybe the method in `authInfo.ts` could be renamed to better reflect the differences.